pkg/csi/cinder/nodeserver.go

```diff
@@ -526,15 +526,24 @@ func (ns *nodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandV
 		return nil, status.Error(codes.InvalidArgument, "Volume ID not provided")
 	}
 	volumePath := req.GetVolumePath()
+	if len(volumePath) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "Volume path not provided")
+	}
 
-	args := []string{"-o", "source", "--first-only", "--noheadings", "--target", volumePath}
-	output, err := ns.Mount.Mounter().Exec.Command("findmnt", args...).CombinedOutput()
+	_, err := ns.Cloud.GetVolume(volumeID)
```

Comment:
we should think of alternative and avoid querying openstack cloud from node , as we can avoid the need of cloud credentials from node. But thats altogether new issue , will be dealt seperately.

Reply:
yes, let's do this at follow up PR